server/src/sylvester/query_resolvers.py

```diff
@@ -0,0 +1,106 @@
+from typing import Callable, Dict, List, Optional
+
+import pymongo
+from db import DB
+from tartiflette import Resolver
+
+from sylvester.event import (Event, LendEvent, RentClaimedEvent, RentEvent,
+                             StopLendEvent, StopRentEvent)
+
+database_name = 'ethereum-indexer'
```

Comment:
we can utilise the config I wrote today if we move the config.py one level up. But you need to rebase this branch on my main first

Reply:
will implement once i merge both the PRs into my fork and then rebase on top of your main.